tasks/76_deep-object-clone/deep-object-clone.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * deepClone - Clones nested object.
+ *
+ * @example
+ * //return { a: 1, b: { c:10 } }
+ * deepClone({ a: 1, b: { c:10 } });
+ * @param {Object} obj - object provided to copying.
+ * @return {Object} new deep copy of object.
+ */
+export function deepClone(obj) {
+  const copied = (function copy(toCopy) {
+    const result = {};
+    for (let key in toCopy) {
+      if (typeof toCopy[key] === 'object' && obj[key] !== null) {
```

Comment:
Let's introduce function outside of deepClone function, in this case we could remove IIFE